projects/microphysics/argo/prog-run-and-eval.yaml

```diff
@@ -0,0 +1,169 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: prognostic-microphysics-
+spec:
+  entrypoint: prognostic-run
+  arguments:
+    parameters:
+      - name: tag
+      - name: config
+      - name: tf_model
+      - name: baseline_tag
+        value: "baseline"
+      - name: on_off_flag
+        value: "--online"
+      - name: do_piggy
+        value: "true"
+  volumes:
+  - name: gcp-key-secret
+    secret:
+      secretName: gcp-key
+  templates:
+  - name: prognostic-run
+    steps:
+    - - name: run-model
```

Comment:
Maybe for later, but I would suggest running a "baseline" run for every model. This will allow us to compare the offline and online performance of a given ML model over the same time period. This workflow will also be more reproducible since the only input data will then be the tf model and fv3config inputs. It should also remove the need for separate baseline and non-baseline configs, since the  `--online/--offline` flag can control this.